tools/sailfish/sailfish.xml

```diff
@@ -56,10 +67,24 @@
             --index $index_path
             #if $single_or_paired.single_or_paired_opts == 'single':
                 --libType ${single_or_paired.strandedness}
-                --unmatedReads ./single.$ext
+                #if $compressed == 'GZ':
+                    --unmatedReads <(zcat ./single.$ext)
+                #else if $compressed == 'BZ2':
+                    --unmatedReads <(bzcat ./single.$ext)
```

Comment:
I would not assume anything ;)
Moreover, Galaxy supports to build containers automatically from these requirements, and then the job is running in a busybox, where this is not installed.